mslice/cli/_mslice_commands.py

```diff
@@ -92,7 +92,7 @@ def GenerateScript(InputWorkspace, filename):
     generate_script(ws_name=workspace_name, plot_handler=plot_handler, filename=filename)
 
 
-def MakeProjection(InputWorkspace, Axis1, Axis2, Units='meV'):
+def MakeProjection(InputWorkspace, Axis1, Axis2, Units='meV', OutputWorkspace=None, Limits=None):
```

Comment:
We don't appear to be using these two arguments?

Reply:
That is tentative as I still am waiting for the solution to the algorithm being run without setting the workspace name

Reply:
Have you found anything that could help with this?

Reply:
Not fully. I think we need to set the name for the `InputWorkspace` argument as well as the pointer variable but I can't see a clean way to do this yet.

Reply:
Is it okay to say that this is what we're waiting for for this pull request to be closed. I have opened an issue to add tests.